homeassistant/components/hassio/strings.json

```
@@ -185,7 +185,7 @@
     },
     "unsupported_docker_version": {
       "title": "Unsupported system - Docker version",
-      "description": "System is unsupported because the wrong version of Docker is in use. Use the link to learn the correct version and how to fix this."
+      "description": "System is unsupported because the wrong version of Docker is in use. For information about the correct version and how to fix this, select Learn more."
```

One small improvement on second sight: "wrong version" sounds a bit weird, Supervisor requires a certain minimum version, so we could go with something like this:

```suggestion
      "description": "System is unsupported because the Docker version is out of date. For information about the correct version and how to fix this, select Learn more."
```